docs/arcaflow/workflows/multi-output.md

```diff
@@ -0,0 +1,9 @@
+# Writing multiple outputs
+
+A common example of two mutually exclusive events could be the availability of your data storage service. Let's assume the service is either available, or unavailable (the unavailble state also includes any states where an error is thrown during data insertion). Multiple workflow outputs allows you to plan for these two events.
+
+In this example, the `success` output collects the data from the specified steps and inserts it into data storage. The `no-indexing` output collects the data, the error logs, and does not store the data.
+
+```yaml
+{!https://raw.githubusercontent.com/arcalot/arcaflow-workflows/multiple-outputs-example/example-workflow/workflow.yaml [ln:90-100]!}
```

Comment:
Is it possible to make it so it just grabs the end of the file, so that the lines aren't incorrect when the source file gets updated?

Reply:
Is this the `mdx-include` functionality in action?

Reply:
yes, this is `mdx-include` in action.

Reply:
The end slice notation should be `[ln:90-e]`, but I have not been able to get it to work.

Reply:
[markdown-include](https://github.com/cmacmackin/markdown-include) is another option, but it does not have an 'EOF' index feature, and it can only include source files on the local filesystem.